repos/keg-cli/src/tasks/docker/tunnel.js

```diff
@@ -0,0 +1,54 @@
+const ngrok = require('ngrok')
+const qrcode = require('qrcode-terminal')
+const { Logger } = require('KegLog')
+
+/**
+ * Create a public tunnel the the url (by default, to `kegdev.xyz`)
+ * @function
+ * @param {Object} args - arguments passed from the runTask method
+ * @param {string} args.command - Initial command being run
+ * @param {Array} args.params - arguments passed from the command line
+ *
+ * @returns {void}
+ */
+const tunnel = async args => {
+  const { params } = args
+  const { qr, source, port } = params
+
+  const ip = process.env.KEG_DOCKER_IP
+
+  Logger.header('Starting tunnel...', 'white')
+
+  const url = await ngrok.connect(source || port || ip)
+
+  Logger.spacedMsg('Source:', Logger.color('magenta', ip))
+  Logger.spacedMsg('Tunnel:', Logger.color('green', url))
+
+  // show a qr code on the terminal that the user can use to quickly access the tunnel on a mobile device
+  qr && qrcode.generate(url, { small: false })
```

Comment:
The qr code is dope. So much so, that I would make it turned on by default. Then you would not need to pass `--qr` everytime

In the `options object` of the `task definition` down below; you can set a `default` for the option.
See my comment in that section to see what I'm saying

Reply:
Yeah works for me. Updated